package.json

```diff
@@ -301,7 +301,6 @@
     "@types/qs": "6.9.7",
     "@types/react": "16.9.52",
     "@types/react-native": "0.63.25",
-    "@types/react-native-dotenv": "0.2.0",
```

Comment:
Removed this because we should encourage ourselves to specify the available values for these instead of falling back to the default `string`. TypeScript will help remind us!